### PR TITLE
Call for Content: 2021/03

### PR DESCRIPTION
**Pull request by domenkozar**
Hi all,

please add Nix news item in form of:

```markdown
- [Example news item title](http://example.com)

  Optional item description.

```

preferably as a PR against `master` branch (or a comment below).

Thank you <3

### CODE REVIEW

**Comment by asymmetric** [comment]
- [Using NixOS as a Router (NaaR)](https://francis.begyn.be/blog/nixos-home-router)
- [My Automagic NixOS Wireguard Setup](https://christine.website/blog/my-wireguard-setup-2021-02-06)

**Comment by domenkozar** [comment]
- [Qubes-lite With KVM and Wayland](https://roscidus.com/blog/blog/2021/03/07/qubes-lite-with-kvm-and-wayland/)
- [nix-script](https://bytes.zone/posts/nix-script/)
- [NixOS on macOS](https://ihpcafe.ihpapp.com/TodaysNixCoffee)
- https://summer.nixos.org/
- https://www.youtube.com/watch?v=QKoQ1gKJY5A&list=PL-saUBvIJzOkjAw_vOac75v-x6EzNzZq-
- https://tech.channable.com/posts/2021-04-09-nix-is-the-ultimate-devops-toolkit.html

**Comment by abcdw** [comment]
- [Flakes in nix 3.0](https://www.youtube.com/watch?v=98EwejpIJzE&list=PLZmotIJq3yOKew30oT8aEbPUOEKBmNpY1)

  Video explains how to improve reproducibility of nix expressions, why and for what we may need flakes and how to use them.

**Comment by boredom101** [comment]
Something I have been working on, that I would like added if it is alright:

[Build arduino sketches like any other nix derivation.](https://github.com/boredom101/nixduino)

File: content/2021/03.md [hunk 1]
```diff
@@ -24,6 +24,10 @@ Description: (TODO: QUOTE)
 
 ## Jobs
 
+- [DevOps Engineer at Secure Compute Startup](https://discourse.nixos.org/t/devops-engineer-at-secure-compute-startup/12008)
+
+  Drive the adoption of Nix at an secure compute startup by developing and maintaining our CI pipeline and infrastructure.
```
Comment (cole-h) [inline content/2021/03.md]:
```suggestion
  Drive the adoption of Nix at a secure compute startup by developing and maintaining our CI pipeline and infrastructure.
```

**Pull request merged**